cluster-api/src/test/java/io/aiven/klaw/clusterapi/services/SchemaServiceTest.java

```diff
@@ -474,19 +482,46 @@ public void getSchemasOfClusterNoSchemas() throws JsonProcessingException {
     assertThat(schemasInfoOfClusterResponse.getSchemaInfoOfTopicList().size()).isEqualTo(0);
   }
 
-  private static ClusterTopicRequest deleteTopicRequest(String topicName) {
-    return ClusterTopicRequest.builder()
-        .clusterName("DEV2")
-        .topicName(topicName)
-        .env("bootStrapServersSsl")
-        .protocol(KafkaSupportedProtocol.SSL)
-        .partitions(1)
-        .replicationFactor(Short.parseShort("1"))
-        .aclsNativeType(AclsNativeType.NATIVE)
-        .deleteAssociatedSchema(true)
-        .schemaClusterIdentification("DEV3")
-        .schemaEnv("schemaservers")
-        .schemaEnvProtocol(KafkaSupportedProtocol.SSL)
-        .build();
+  @Test
+  @Order(13)
+  public void getSchemaRegistryStatusClusterStatusOnline() throws JsonProcessingException {
+    String environmentVal = "ENVIRONMENT";
+    KafkaSupportedProtocol protocol = KafkaSupportedProtocol.PLAINTEXT;
+    String clusterIdentification = "CLUSTER";
+    String suffixUrl = environmentVal + "/" + "subjects";
+    String url = "http://" + suffixUrl;
+
```

Comment:
If you can make these constants as well

String environmentVal = "ENVIRONMENT";
KafkaSupportedProtocol protocol = KafkaSupportedProtocol.PLAINTEXT;
String clusterIdentification = "CLUSTER";
String suffixUrl = environmentVal + "/" + "subjects";
String url = "http://" + suffixUrl;

Reply:
Using constants from TestConstants class and adding a class constant for this URL string.